### PR TITLE
Add edit toggle to company view and remember the changes made.

### PR DESCRIPTION
**Pull request by ztolley**

### CODE REVIEW

File: stylesheets/_shame.scss [hunk 1]
```diff
@@ -17,6 +17,10 @@
   text-align: right;
 }
 
+.actions--left {
+  text-align: left;
```
Comment (teneightfive) [inline stylesheets/_shame.scss]: Is this needed? The default `text-align` is left.

Comment (teneightfive) [inline stylesheets/_shame.scss]: Ah. Spacing. Maybe we should find a better way to do the spacing.

Comment (ztolley) [inline stylesheets/_shame.scss]: yeah.. we should... later

File: views/includes/company-details.jade [hunk 2]
```diff
@@ -25,25 +25,55 @@ table.table-detail
...
+                  a(href="#") #{experiance.year} - #{experiance.country}
+        tr
+          th Countries of interest
+          td
+            if company.countryOfInterest
+              each countryOfInterest in company.countryOfInterest
+                div= countryOfInterest
+
+    p.actions
+      button.button.button-secondary.edit-button Edit Company
```
Comment (teneightfive) [inline views/includes/company-details.jade]: Can we keep language consistent and use lowercase company

File: views/includes/company-details.jade [hunk 3]
```diff
@@ -25,25 +25,55 @@ table.table-detail
...
+                div
+                  a(href="#") #{experiance.year} - #{experiance.country}
+        tr
+          th Countries of interest
+          td
+            if company.countryOfInterest
+              each countryOfInterest in company.countryOfInterest
+                div= countryOfInterest
+
+    p.actions
```
Comment (teneightfive) [inline views/includes/company-details.jade]: Should this be using the left aligned actions modifier

Comment (teneightfive) [inline views/includes/company-details.jade]: Had a quick look and might be able to drop that `.actions--left` class and add bottom margin to the details table?

File: controllers/CompanyViewController.js [hunk 4]
```diff
@@ -31,13 +42,42 @@ function get(req, res) {
...
+function post(req, res) {
+  let companyNum = req.params.id;
+
+  uktiCompanyData[companyNum] = {
+    website: req.body.website,
+    businessDescription: req.body.businessDescription,
+    countryOfInterest: req.body.countryOfInterest
+  };
+
+  res.redirect(`/companies/${companyNum}`);
```
Comment (teneightfive) [inline controllers/CompanyViewController.js]: Can we include the query param in the redirect if it exists. Otherwise it breaks the back to companies link

**Pull request merged**